Elsa.sln

```diff
@@ -1,25 +1,26 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
+# 
```

Comment:
?

Reply:
Not sure, looks like dotnet cli added. Removed it